### ISSUE TITLE
[BUG] Skull of Impending Doom's self damage isn't reduced by Armor (Era/PTR)

### ISSUE THREAD

**Issue by perpabc** [JamminL/wotlk-classic-bugs#710]
#### 1.1. Description
Skull of Impending Doom deals Physical damage to the user. The damage should be reduced by Armor. 
  #### 1.2. How to Reproduce
           1. .additem 4984
           2. Look at your Armor value 
           3. Use Skull until you hit a threshold where the the return damage ignores your Armor (maximum self damage is 500, in my clip, Skull is doing 435 damage per tick, and I have 30% Physical damage reduction. With the highest roll of Skull self damage, I should be taking a maximum of 350 damage per tick.)
  #### 1.3  Source Material
https://gyazo.com/402d169c9e273fdf426cf6b2eb22a63f
#### 2.1. Description
Armor should reduce the damage taken by Skull of Impending Doom
  #### 2.2 Source Material
Impossible to find material on this as you can't prove a negative - but we know that Skull deals Physical damage: https://www.wowhead.com/classic/spell=5024/flee and Armor reduces Phys damage taken. Reference client may be helpful here.

**Comment by knasen88** [comment]
since when would a dot be reduced by armor?
if anything you gotta find a source where it's ticking for less than the minimum amount (without any -damage reduction effects rolling)

**Comment by perpabc** [comment]
Yeah that's what I'm looking for, difficult to find a source where it's discussed, or shown in a video sadly

**Comment by Rowain3** [comment]
Start at 7:19
[https://www.youtube.com/watch?v=HUPexEfCG7g](url)
This is an old swifty PvP video where he showcases using Skull of Impending Doom. Upload date is August 19, 2006 which means this was Patch 1.11.2. However, there are no notes in the Patch 1.12 series indicating there was any change to Skull of Impending Doom. 

The video does not showcase any ticking values of less than 100, but they appear to have the exact same damage tick upon activation and consistently hover ~250 with variations as he uses different gear pieces. I do not know the full implication of this data, but hopefully this video evidence helps.
